manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml

```diff
@@ -215,6 +231,10 @@ spec:
             postgresqlCpuRequest:
               description: 'PostgreSQL deployment CPU request (default: no request)'
               type: string
+            postgresqlImage:
+              description: Image string used for the postgresql deployment By default
```

Comment:
The pattern elsewhere is `<description> (Default: <default value>)`, so can we change it to that for consistency?  If not, there should at least be a period or comma between the description and `By default`.

Reply:
both good, because the default is based on the other parameters, I didn't have a value.  would something like `description (default: <postgresqlImageName>:<postgresqlImageTag>)` work ?